src/rprblender/engine/viewport_engine_2.py

```diff
@@ -178,13 +156,13 @@ def render_update(progress):
                     self.rpr_context.set_render_update_callback(render_update)
                     is_set_callback = True
 
-                # rendering
+                # Rendering
```

Comment:
What idea behind of fixing comments lowercase to uppercase?

Reply:
not any )